### PR TITLE
feat(Task-19): As an User, I want to know what I'm about to do

### PR DESCRIPTION
**Pull request by jLagosCarrera**

### CODE REVIEW

**Review by gbtec-ext-jorgevillar — APPROVED** [review]

File: src/app/search-page/searchPage.controller.js [hunk 1]
```diff
@@ -25,6 +28,30 @@ export default class SearchPageController {
...
+                console.log(answer);
+                this.$mdToast.show(
+                    this.$mdToast.simple()
+                    .textContent('We will contact you soon!')
+                    .position('bottom right')
+                    .theme('success-toast')
+                    .hideDelay(3000)
+                );
+            }).catch(() => {
+                //Do nothing on dialog cancel
```
Comment (eloydotco) [inline src/app/search-page/searchPage.controller.js]: so is not necessary the .catch() here

File: src/app/shared/components/displayable-list/displayableList.component.html [hunk 2]
```diff
@@ -1,14 +1,24 @@
-<md-icon class="material-icons recent" ng-show="$ctrl.useHeader">{{ $ctrl.displayIcon }}</md-icon>
+<md-icon class="material-icons recent" ng-show="$ctrl.useHeader">
+    <md-tooltip md-delay="400">
+        {{ $ctrl.listType }} cities list
```
Comment (eloydotco) [inline src/app/shared/components/displayable-list/displayableList.component.html]: you can use an ont-time binding here

File: src/app/shared/components/displayable-list/displayableList.component.html [hunk 3]
```diff
@@ -1,14 +1,24 @@
...
+    </md-tooltip>
+    {{ $ctrl.displayIcon }}
```
Comment (eloydotco) [inline src/app/shared/components/displayable-list/displayableList.component.html]: and also here

File: src/app/shared/components/displayable-list/displayableList.component.html [hunk 4]
```diff
@@ -1,14 +1,24 @@
...
+</md-icon>
 <ul class="flex-row justify-space-around flex-wrap remove-all-spacing">
     <li class="flex-row justify-center align-center flex-number"
         ng-repeat="city in $ctrl.listToDisplay track by $index">
         <a class="link-effect spacing-4" href ng-click="$ctrl.search(city, $ctrl.latestSearches)">
+            <md-tooltip md-delay="400">
+                {{ $ctrl.listType }} city
```
Comment (eloydotco) [inline src/app/shared/components/displayable-list/displayableList.component.html]: and here :D

**Review by eloydotco — APPROVED** [review]

**Pull request merged**